specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -256,34 +262,34 @@
           {
             "name": "resourceGroupName",
             "in": "path",
-            "description": "Azure resource group name",
+            "description": "The name of the resource group that contains the container instance.",
             "required": true,
             "type": "string"
           },
           {
             "name": "containerName",
             "in": "path",
-            "description": "Container name",
+            "description": "The name of the container instance.",
             "required": true,
             "type": "string"
           },
           {
             "name": "containerGroupName",
             "in": "path",
-            "description": "Container group name",
+            "description": "The name of the container group the container instance belongs to.",
             "required": true,
             "type": "string"
           },
           {
             "name": "tail",
             "in": "query",
-            "description": "Only show this number of log lines. If not provided, all available logs are shown.",
+            "description": "The number of lines to show from the container instance log. If not provided, all available logs are shown.",
```

Comment:
Maybe explicitly say about `tail`? Also if not provided, I think it will show up to 4mb logs. No more than that.

Reply:
Good feedback - added a bit about logs coming from the tail and the 4mb limit